extensions/opentelemetry/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/TracerRecorder.java

```diff
@@ -31,19 +35,38 @@
     }
 
     /* STATIC INIT */
-    public RuntimeValue<SdkTracerProvider> createTracerProvider(String serviceName, String serviceVersion,
+    public RuntimeValue<SdkTracerProvider> createTracerProvider(TracerConfig tracerConfig,
+            String serviceName,
+            String serviceVersion,
             ShutdownContext shutdownContext) {
-        BeanManager beanManager = Arc.container().beanManager();
+        ArcContainer container = Arc.container();
+        BeanManager beanManager = container.beanManager();
+
+        InjectableInstance<IdGenerator> idGenerator = container.select(IdGenerator.class);
```

Comment:
What does `InjectableInstance` offer that `Instance` doesn't support for what we need?

Reply:
Nothing really, changed to `Instance`